frontend/src/components/base/Button/Button.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render } from '../../../test-utils';
+import { Button } from './Button';
+
+describe('Button', () => {
+  const mockClickHandler = jest.fn();
```

Comment:
Can you move this into the `it`'s where it's being used?

Reply:
Done.